Doc/library/argparse.rst

```diff
@@ -892,14 +663,7 @@ how the command-line arguments should be handled. The supplied actions are:
 * ``'store_true'`` and ``'store_false'`` - These are special cases of
   ``'store_const'`` used for storing the values ``True`` and ``False``
   respectively.  In addition, they create default values of ``False`` and
-  ``True`` respectively.  For example::
```

Comment:
I feel this example is probably helpful to keep. Not everyone is going to read the tutorial, and examples are extremely useful for people just glancing across the docs page to find the thing they need.